charts/spire/templates/agent-daemonset.yaml

```diff
@@ -105,7 +111,17 @@ spec:
           args: [
             "-csi-address", "/spiffe-csi/csi.sock",
             "-kubelet-registration-path", "/var/lib/kubelet/plugins/csi.spiffe.io/csi.sock",
+            "-http-endpoint", ":9809",
```

Comment:
Could you explain this feature? Also maybe isolate that in separate PR.

Reply:
This flag adds a livenessProbe to the node-driver-registrar and goes with livenessProbe definition I added a few lines later. My company likes to turn on liveness probes whenever possible, but this change is entirely optional.